src/App/components/EmojiInput/index.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import "./styles.css";
+
+const EmojiInput = props => {
+  const { input, parent, selected, onClick } = props;
```

Comment:
Are you up for destructuring this in the argument?

```js
const Emoji = ({ input, parent, selected, onClick }) => {
```

Reply:
yes, will do.